app/controllers/admin/volunteers_controller.rb

```diff
@@ -4,6 +4,8 @@ def index
       @nav = "Volunteers"
       @volunteers = User.where(role: :volunteer).order(:last_name)
 
+      session[:filters] = {volunteers_table: {}} unless @stimulus_reflex
```

Comment:
Under what conditions with `@stimulus_reflex` be falsy?

Reply:
Ideally when the page loads for the first time

https://docs.stimulusreflex.com/rtfm/persistence#the-stimulus_reflex-instance-variable

Reply:
Makes sense. Thanks for the link!